src/vs/workbench/api/common/extHostApiCommands.ts

```diff
@@ -406,6 +406,22 @@ const newCommands: ApiCommand[] = [
 		],
 		ApiCommandResult.Void
 	),
+	// --- type hierarchy
+	new ApiCommand(
+		'vscode.prepareTypeHierarchy', '_executePrepareTypeHierarchy', 'Prepare type hierarchy at a position inside a document',
```

Comment:
For call hierarchy, I find reference-view extension executes `vscode.prepareCallHierarchy` to fetch required data. 
But in my experiment, I failed to execute the command by `id`, but managed to execute it by `internalId`. I haven't yet taken a deep look at it. Please let me know if I wrongly used it.

Reply:
hm... hard to say with an error message. this looks all good so far

Reply:
ok, for the moment I just leave it as is. I can test it later when implementing other parts.